wpilib/wpilib/ultrasonic.py

```diff
@@ -59,6 +59,8 @@ class Unit:
     #: ultrasonic sensor list
     sensors = weakref.WeakSet()
 
+    _sensor_mutex = threading.Lock()
```

Comment:
This name is confusing. This only guards the `sensors` variable. Shouldn't it be named after said variable?